### PR TITLE
FSR-705 Accessibility Changes

### PR DESCRIPTION
**Pull request by LeeGordon83**
https://eaflood.atlassian.net/browse/FSR-705
https://eaflood.atlassian.net/browse/FSR-701
https://eaflood.atlassian.net/browse/FSR-886

### CODE REVIEW

File: server/models/views/station.js [hunk 1]
```diff
@@ -385,6 +354,80 @@ class ViewModel {
...
+    let telemetryData
+    if (telemetry.length) {
+      telemetryData = telemetryForecastBuilder(this.telemetry, forecastData, this.station.type)
+    }
+    this.telemetryRefined = telemetryData || []
+  }
+}
+
+function getBannerIcon (id) {
+  return severity.filter(item => item.id === id)[0].icon
```
Comment (maxcbc) [inline server/models/views/station.js:391]: You could use `severity.find(item => item.id === id)?.icon` here

File: server/src/sass/components/_line-chart.scss [hunk 2]
```diff
@@ -1,221 +1,315 @@
...
-    .threshold--alert {
-        .threshold__line {
-            stroke: $govuk-border-colour;
-        }
-    }
-    .threshold.threshold--selected.threshold--alert,
-    .threshold.threshold--mouseover.threshold--alert {
-        .threshold__line {
-            stroke: $govuk-text-colour;
+    // .threshold--alert {
```
Comment (maxcbc) [inline server/src/sass/components/_line-chart.scss:281]: Does this need to be commented or can it just be removed

File: server/src/sass/components/_flood-status.scss [hunk 3]
```diff
@@ -39,61 +64,89 @@
...
+    &:after {
+        color: govuk-tint(govuk-colour('blue'), 90%);
+        border-color: currentColor;
+    }
+    svg {
+        vertical-align: middle;
+        color: white;
+        width: 20px;
+        height: 20px;
+        margin: 0px 5px;
```
Comment (maxcbc) [inline server/src/sass/components/_flood-status.scss:120]: `0px` can be `0`

File: server/src/sass/components/_flood-status.scss [hunk 4]
```diff
@@ -39,61 +64,89 @@
...
+        top: 0px;
+        bottom: 0px;
+        right: 0px;
+        border: 1px solid transparent;
+        pointer-events: none;
     }
 }
 .defra-flood-status-item--severe {
-    background-color: mix(white, #e3000f, 93%);
+    background-color: govuk-tint(#e3000f, 93);
```
Comment (maxcbc) [inline server/src/sass/components/_flood-status.scss:85]: hex colours in here should probably be extracted to sass variables to be more DRY

File: server/src/sass/components/_flood-status.scss [hunk 5]
```diff
@@ -39,61 +64,89 @@
...
-        @include mq ($from: tablet) {
-            background-size: 100px 250px;
-            width: 50px;
-            height: 50px;
-        }
+    &:after {
+        position: absolute;
+        box-sizing: border-box;
+        content: '';
+        left: 0px;
```
Comment (maxcbc) [inline server/src/sass/components/_flood-status.scss:76]: `0` doesn't need units

**Review by maxcbc — DISMISSED** [review]

**Comment by maxcbc** [comment]
Code looks fine other than some small finickity bits. Have not pulled down and checked with screen reader. When this PR is ready I'll check with screen reader.

**Review by nikiwycherley — APPROVED** [review]

**Pull request merged**